db/migrate/20160925225618_set_guid_not_null_on_users.rb

```diff
@@ -0,0 +1,6 @@
+class SetGuidNotNullOnUsers < ActiveRecord::Migration
+  def change
+    User.destroy_all
```

Comment:
No, it's not a mistake.

If your dev box has existing users, this migration will fail. This project isn't deployed to any staging  or production servers, so no real data can be lost this way.